src/db/migrations/20240320074757_initial_migration.ts

```diff
@@ -65,12 +71,22 @@ export async function up(knex: Knex): Promise<void> {
         table.timestamp("created_at").defaultTo(knex.fn.now());
         table.text("verdict");
         table.integer("raw_score");
-        table.integer("running_time_ms");
-        table.integer("running_memory_byte");
-        table.integer("compile_time_ms");
-        table.integer("compile_memory_byte");
+        table.integer("running_time_ms").unsigned();
+        table.integer("running_memory_byte").unsigned();
+        table.integer("compile_time_ms").unsigned();
+        table.integer("compile_memory_byte").unsigned();
         table.index(["submission_id", "created_at"]);
       })
+      .createTable("contests", (table) => {
+        table.uuid("id").primary().defaultTo(knex.raw("uuid_generate_v4()"));
+        table.text("slug").notNullable().unique();
+        table.text("title").notNullable();
+        table.text("description");
+        table.uuid("owner_id").notNullable().references("id").inTable("users");
+        table.timestamp("start_time").notNullable().defaultTo(knex.fn.now());
```

Comment:
Please make this nullable (no start time == contest not started and will never start).

Sometimes we just like making contests before setting a date. We often just set the date 100 years into the future, but since we're here and we have the freedom...